client/src/components/RatingsAndReviews/ReviewsList/ReviewsList.scss

```diff
@@ -0,0 +1,3 @@
+.reviews_section .addReviews {
+  margin-top: 20px;
+}
```

Comment:
Could you please add a new line at the end of the file? Just a tiny best practice thing.

Reply:
done.